### ISSUE TITLE
Add context to all methods

### ISSUE THREAD

**Issue by hbagdi** [hbagdi/go-unsplash#31]
If anyone is using the package, this would break all those.

**Comment by terinjokes** [comment]
I'd suspect most users who are using this seriously would be vendoring dependencies. The downside is you haven't tagged any releases, so they're using "master" and not semver.

**Comment by hbagdi** [comment]
Yes, I should have version it long back. Better late than never though. Let me version it now and put it on gopkg.in

**Comment by terinjokes** [comment]
I wouldn't recommend gopkn, because of how it messes with import paths.
Just use git tags and tools like dep and glide will use it without any
extra work.
